yaas/parsers/host.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import host
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
+  parser = parent_parser.add_parser(
+      'host',
+      help=inspect.getdoc(host))
+
+  subcommands = [
+      host_ls,
+      host_show,
+      ]
+
+  subparsers = parser.add_subparsers()
+  for subcommand in subcommands:
+      subcommand(subparsers)
+
+def host_ls(parent_parser):
+    def ls(args):
```

Comment:
I'm not really a fan of this nested functions technique, but overall the good in this PR decimates the bad. We can work on improving this in subsequent PRs.

Namely, how do I access just that function from outside its nest? That could be desirable, and I suspect it is achievable using classes instead.